backend/packages/Upgrade/src/api/services/StratificationService.ts

```diff
@@ -22,8 +22,12 @@ export class StratificationService {
   ): FactorStrata[] {
     const formattedResults = results.reduce((formatted, result) => {
       const { factor, value, count, experimentIds } = result;
+      const expIds: string[] = [];
+      experimentIds.forEach((expId) => {
+        expId ? expIds.push(expId) : null;
```

Comment:
This is a long way of doing a `map`, but why exactly are we setting falsey values to `null`? Do we want `null` for some special reason somewhere else?

Reply:
It's a bit confusing to use a `forEach` and a ternary operator that is returning null to nowhere, all we're trying to do is a normal filter. I think `const expIds = experimentIds.filter(experimentId => !!experimentId)` works fine, right?